tests/lvmo/test_lvm_alerts.py

```diff
@@ -0,0 +1,233 @@
+import time
+import logging
+
+import pytest
+
+from ocs_ci.framework.pytest_customization.marks import (
+    tier1,
+    skipif_lvm_not_installed,
+)
+from ocs_ci.ocs import constants
+from ocs_ci.framework.testlib import skipif_ocs_version, ManageTest
+from ocs_ci.ocs.cluster import LVM
+from ocs_ci.ocs.node import get_worker_nodes
+from ocs_ci.utility.prometheus import PrometheusAPI
+
+log = logging.getLogger(__name__)
+new_prom = PrometheusAPI()
```

Comment:
Please use method scope fixture to share this object. You cannot have it here as it's failing collection of the tests because that time the cluster doesn't exists yet.